rusregistry/entities/individual_enterpreneur.py

```diff
@@ -0,0 +1,31 @@
+from entities.entity import Entity, get_unique_id
+from utils.column_mapping import IE_MAPPING
+import pandas as pd
+from csv_processing.utils import fix_inn
+
+class IndividualEnterpereneur(Entity):
```

Comment:
More general note: individual entrepreneurs are not that interesting in terms of asset tracing most of the time, so I'd treat them as a low priority.